app/routes/goal_routes.py

```diff
@@ -1 +1,75 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, abort, make_response
+import requests
+import os
+from .route_utilities import validate_model, create_model
+from ..models.goal import Goal
+from ..models.task import Task
+from ..db import db
+
+
+bp=Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+@bp.get("")
+def get_goals():
+    query=db.select(Goal)
+    goals = db.session.scalars(query.order_by(Goal.id))
+    goals_response=[]
+    for goal in goals:
+        goals_response.append(goal.to_dict())
+    return goals_response
+
+@bp.get("/<id>")
+def get_one_goals(id):
+    goal = validate_model(Goal, id)
+    return {"goal": goal.to_dict()},200 
```

Comment:
Nitpick: missing space after comma

The default status code Flask returns to the client is 200. Therefore we do not need to return it ourselves explicitly. 

Note that you don't explicitly return 200 for the `get_goals` and `get_one_goal_tasks` routes. 

I'd prefer not to explicitly return a status code if I don't need to because it adds more code and feels redundant because the framework does it for us. If you choose to return 200 or choose not to, just be consistent in the choice and apply the decision throughout the entire codebase, instead of sometimes returning it and sometimes not.

```suggestion
    return {"goal": goal.to_dict()}
```